Source/UI/NewUI/PaymentForm/Factory/Theme/Theme.swift

```diff
@@ -0,0 +1,81 @@
+//
+//  Theme.swift
+//  
+//
+//  Created by Alex Ioja-Yang on 08/08/2022.
+//
+
+import UIKit
+
+/// Template theme generating UI components for your layout
+public struct Theme {
+
+    // MARK: Font colors
+    /// Font color associated with text displaying primary font colour, like inputs, titles and other important text
+    public var primaryFontColor: UIColor
+    /// Font color associated with text displaying secondary information, like subtitles and other non core text
+    public var secondaryFontColor: UIColor
+    /// Font color associated with text displayed inside actionable buttons
+    public var buttonFontColor: UIColor
+    /// Font color associated with text
+    public var errorFontColor: UIColor
+
+    // MARK: Font sizes
+    /// Font size used for page headers
+    public var headerFontSize = UIFont.preferredFont(forTextStyle: .title1).pointSize
+    /// Font size used for input sections titles
+    public var titleFontSize = UIFont.preferredFont(forTextStyle: .subheadline).pointSize
+    /// Font size used for text displaying secondary information, like subtitles and other non core text
+    public var subtitleFontSize = UIFont.preferredFont(forTextStyle: .footnote).pointSize
+    /// Font size used for input fields where user is inputting the values
+    public var inputFontSize = UIFont.preferredFont(forTextStyle: .subheadline).pointSize
+    /// Font size used for buttons that the user will interact with
+    public var buttonsFontSize = UIFont.preferredFont(forTextStyle: .body).pointSize
+
+    // MARK: UI Formatting colors
+    /// Color used as background on screens
+    public var backgroundColor: UIColor
+    /// Color used as background for text input fields
+    public var textInputBackgroundColor: UIColor = .clear
+    /// Color used for border on text input field when its focussed
+    public var focussedTextInputBorderColor: UIColor = .clear
+    /// Color used for border around data input sections. Will require a borderWidth to be provided for border to be shown
+    public var borderColor: UIColor = .clear
+    /// Color used for border around text inputs. Will require a textInputBorderRadius to be provided for the border to be shown
+    public var textInputBorderColor: UIColor = .clear
+    /// Color used for border when a validated field is receiving an error. Will be used by errorBorder or errorTextInputBorder as setup
+    public var errorBorderColor: UIColor
```

Comment:
```suggestion
    public var errorBorderColor: UIColor = .clear
```

Reply:
I was tempted to default it to clear, but highlighting errors doesn't work well without the highlighting part. This is also why unlike all the other border related defaults, `errorBorderRadius` and `errorBorderWidth` are not 0!

We want to encourage consumer to highlight key UX element by default!